lib/calendar.ex

```diff
@@ -12,6 +12,7 @@ defmodule Calendar do
       alias Calendar.TimeZoneData
       alias Calendar.TzPeriod
       alias Calendar.Strftime
+      alias Calendar.Interval
```

Comment:
Oh, sorry I hadn't seen your last commit and was just looking at this line, thinking it wasn't change in the module itself. Could you change this to `Calendar.DateTime.Interval` ? Then I think this PR will be good to go.